### DISCUSSION TITLE
ui branch use Node18   build is very slow and the bundle size is huge

### DISCUSSION THREAD

**Discussion by DreamPWJ** [arthelokyo/astrowind#185] [General]
![image](https://user-images.githubusercontent.com/16730031/217970734-13588116-ae54-420c-b308-ae11e46c71fa.png)

**Comment by DreamPWJ** [comment]
`npm WARN deprecated core-js@2.6.12: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js. `

**Comment by DreamPWJ** [comment]
sharp: Installation error: socket hang up

**Comment by DreamPWJ** [comment]
And I hope that the header navigation will not be hidden, and it can be configured to hide or not  @prototypa

**Comment by prototypa** [comment]
Hi @DreamPWJ, I'll write to you soon to see the problems in detail. Thank you

**Comment by DreamPWJ** [comment]
![568edecbc9349985feaaa7327f96844](https://user-images.githubusercontent.com/16730031/218225913-8dd11042-9867-4cd2-9965-bc141817f4b7.jpg)
There is a style problem in the Android WebView  lower version of the mobile phone kernel  @prototypa

**Comment by DreamPWJ** [comment]
![image](https://user-images.githubusercontent.com/16730031/218225834-1c85066d-1f88-4a4b-9269-47ef31b98e1b.png)
Does not support dark mode

**Comment by DreamPWJ** [comment]
googleAnalyticsId vs  googleSiteVerificationId What's the difference, the different configuration files of the distribution， configuration does not take effect

**Comment by DreamPWJ** [comment]
SEO has many key descriptions, where should I configure them? @prototypa

**Comment by DreamPWJ** [comment]
When I default to the dark theme, the light theme is loaded for the first time and then switched to the dark theme @prototypa

**Comment by prototypa** [comment]
Hello @DreamPWJ
We are super busy with something, soon I will answer all these questions in detail. Thank you.
